developer-docs/account/README.md

```diff
@@ -1,358 +1,204 @@
+The Account Service simplifies the management of user accounts, defined by a user's handle and MSA Id (Message Source Account Identifier) on the [Frequency](https://docs.frequency.xyz/) blockchain using [DSNP](https://dsnp.org/) protocols.
 
-Modify any environment variables in the `.env` file as needed. The complete set of environment variables is documented [here](./ENVIRONMENT.md), and a sample environment file is provided [here](../../env-files/account.template.env).
+## 🔍 Architecture Overview <a name="architecture-overview"></a>
 
-1. Copy the template values into the .env.account file.
+The Account Service is built using NestJS and is divided into two main components:
 
-```sh
-   cp env-files/account.template.env .env.account
-```
+1. **API**: Handles incoming HTTP requests
+2. **Worker**: Processes jobs that require blockchain interaction
 
-2. Replace template values with values appropriate to your environment.
+![Gateway Account Service Architecture](./account_service_arch.drawio.png)
 
-### Install
+## 🔑 Key Features <a name="key-features"></a>
 
-Install NPM Dependencies:
+- Account creation using [SIWF](https://github.com/ProjectLibertyLabs/siwf) (Sign-In With Frequency)
+- User and Provider Account management
+- Handle (username) claiming and management
+- Key management (adding and retrieving keys)
+- Delegation info retrieval for MSA Id
+- Seamless integration with Web3 wallets
 
-```sh
-  npm install
-```
+## 💻 Getting Started <a name="getting-started"></a>
 
-### Usage
+This section will guide you through setting up the Account Service for both quick start and local development.
 
-Note: using [docker compose file](../../docker-compose.yaml) to start the services. This will start the services in development mode.
+### Prerequisites
 
-The following command will start all of the necessary containers for the account service to run in development mode.
+Ensure you have the following installed:
+- [Node.js](https://nodejs.org)
+- [Docker](https://docs.docker.com/get-docker/)
 
-```bash
-./scripts/account/restart-chain-docker.sh
-```
+### Quick Start
 
-In order to run the `account-service` in development mode without containers, you can use the following commands:
+1. Clone the repository and navigate to the project directory:
+   ```bash
+   git clone https://github.com/ProjectLibertyLabs/gateway.git
+   cd gateway
+   ```
 
-#### 1. Start the Redis server container and the Frequency container. You can view the logs with your Docker setup
+2. Copy the environment template:
+   ```bash
+   cp env-files/account.template.env .env.account
+   ```
 
-```bash
-./scripts/account/restart-local-dev.sh
-```
+3. Install dependencies:
+   ```bash
+   npm install
+   ```
 
-Once [Frequency](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/explorer) is up, you can monitor the transactions on the blockchain.
+4. Start all services using Docker:
+   ```bash
+   ./scripts/account/restart-chain-docker.sh
+   ```
```

Comment:
I think this is the correct command, but I haven't tested in some time.

```suggestion
4. Start supporting services using Docker:
   ```bash
   ./scripts/account/restart-chain-docker.sh
   ```
5. Start account service api and worker applications using Docker:
   ```bash
   docker compose -f docker-compose.yaml -f docker-compose-e2e.account.yaml --profile account up -d
```